src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export function Person({ person: { name, age, sex, isMarried, partnerName } }) {
+  const spouse = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {!!age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {`${partnerName} is my ${spouse}`}
+          </p>
+        )
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
I would do the ternary operator inside of `p` tag and only change text inside of it.

What benefits may come from this? For example: When the className for the tag changes, you don't have to correct it in two places 

Something like this:
```suggestion
<p className="Person__partner">
      {isMarried
        ? `${partnerName} is my ${spouse}`
        : 'I am not married'
      }
      </p>
```